tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Users/Permissions/ContentStartNodes.spec.ts

```diff
@@ -0,0 +1,100 @@
+﻿import {ConstantHelper, test} from '@umbraco/playwright-testhelpers';
+import {expect} from '@playwright/test';
+
+const testUser = {
+  name: 'Test User',
+  email: 'verySecureEmail@123.test',
+  password: 'verySecurePassword123',
+};
+
+const userGroupName = 'TestUserGroup';
+
+const rootDocumentTypeName = 'RootDocumentType';
+const childDocumentTypeOneName = 'ChildDocumentTypeOne';
+const childDocumentTypeTwoName = 'ChildDocumentTypeTwo';
+let childDocumentTypeOneId = null;
+let childDocumentTypeTwoId = null;
+let rootDocumentTypeId = null;
+
+let testUserCookieAndToken = {cookie: "", accessToken: "", refreshToken: ""};
+
+let rootDocumentId = null;
+let childDocumentOneId = null;
+const rootDocumentName = 'RootDocument';
+const childDocumentOneName = 'ChildDocumentOne';
+const childDocumentTwoName = 'ChildDocumentTwo';
+
+let userGroupId = null;
+
+test.beforeEach(async ({umbracoApi}) => {
+  await umbracoApi.documentType.ensureNameNotExists(rootDocumentTypeName);
+  await umbracoApi.documentType.ensureNameNotExists(childDocumentTypeOneName);
+  await umbracoApi.documentType.ensureNameNotExists(childDocumentTypeTwoName);
+  await umbracoApi.user.ensureNameNotExists(testUser.name);
+  await umbracoApi.userGroup.ensureNameNotExists(userGroupName);
+
+  childDocumentTypeOneId = await umbracoApi.documentType.createDefaultDocumentType(childDocumentTypeOneName);
+  childDocumentTypeTwoId = await umbracoApi.documentType.createDefaultDocumentType(childDocumentTypeTwoName);
+  rootDocumentTypeId = await umbracoApi.documentType.createDocumentTypeWithAllowedTwoChildNodes(rootDocumentTypeName, childDocumentTypeOneId, childDocumentTypeTwoId);
+  rootDocumentId = await umbracoApi.document.createDefaultDocument(rootDocumentName, rootDocumentTypeId);
+  childDocumentOneId = await umbracoApi.document.createDefaultDocumentWithParent(childDocumentOneName, childDocumentTypeOneId, rootDocumentId);
+  await umbracoApi.document.createDefaultDocumentWithParent(childDocumentTwoName, childDocumentTypeTwoId, rootDocumentId);
+
+  userGroupId = await umbracoApi.userGroup.createSimpleUserGroupWithContentSection(userGroupName);
+});
+
+test.afterEach(async ({umbracoApi}) => {
+  // Ensure we are logged in to admin
+  await umbracoApi.loginToAdminUser(testUserCookieAndToken.cookie, testUserCookieAndToken.accessToken, testUserCookieAndToken.refreshToken);
+  await umbracoApi.documentType.ensureNameNotExists(rootDocumentTypeName);
```

Comment:
Add steps to delete the input documents also.

Reply:
The Documents are deleted when I delete the DocumentTypes